README.md

```diff
@@ -189,6 +192,7 @@ Where to discover new MATLAB resources.
 * [MATLAB Style Guide 1](http://note.sonots.com/Matlab/MatlabCodingStyle.html)
 * [MATLAB Style Guide 2](http://4dpiecharts.com/matlab-code-style-guide/)
 * [Octave and MATLAB Snippets](https://git.io/mtips) - Collection of code snippets
+* [knkutils](https://github.com/kendrickkay/knkutils) - MATLAB utility functions written by Kendrick Kay 
```

Comment:
I find "choose" (programming) useful and if I would use MATLAB I would like "vflatten" (Octave has "vec"). I could see a few more inside anonymous functions.